src/pages/UserPage/UserPage.tsx

```diff
@@ -0,0 +1,180 @@
+        [1,2,3,4,5].map(item => {
+          return (
+            <SwiperSlide>
+              <div className='Test'>
+                <div className='Test-title'>QA Manual</div>
+                <div className='Test-result'>
+                  <div className='Bar'>
+                    <div className='Bar-fill'></div>
+                    <div className='Bar-procent'>30%</div>
+                    <div className='Bar-done'>{iconDone}</div>
+                  </div>
+                  <div className='Test-result--date'>тест от 4 января</div>
+                </div>
+                <div className='Test-message'>
+                  <div>Rare.</div>
+                  <div>Это только начало пути!</div>
+                  <div>Прожарь свои знания с нами.</div>
+                </div>            
+              </div>
+            </SwiperSlide>
+          )
+        })
+        }
+      </Swiper>
+      <button className='SwiperArrow swiper1-button-next' ref={nextElRef}>{arrowSlider}</button>
+    </div>
+      </section>
+      <section className='Recomendation'>
+        <div className='container'>
+          <div className='Recomendation-Title'>Полезные материалы для прожарки знаний</div>
+          <div className='SwiperWrapper'>          
+            <button className='SwiperArrow' ref={prevElRef2}>{arrowSlider}</button>
+            <Swiper
+            className='Swiper2'
+            navigation={{
+              prevEl2,
+              nextEl2,
+            }}
+            modules={[Navigation, Pagination]}
+            effect={"cube"}
+            pagination={true}
+            loop={true}
+            spaceBetween={64}
+            slidesPerView={1}
+            breakpoints={{
+              700: {
+                slidesPerView: 2
+              },
+              1000: {
+                slidesPerView: 3
+              }
+            }}
+          >            
+            {
+              [1,2,3,4,5].map(item => {
+                return (
+                  <SwiperSlide>
+                    <div className='Recomendation-Card'>
+                      <div className='Recomendation-Card--Title'>Java без опыта!</div>
+                      <div className='Recomendation-Card--Description'>
+                        <div>В Java-программирование можно прийти практически из любой сферы: это подтверждают результаты опроса. Для начала мы решили узнать, чем наши</div>
+                        <div className='IconRecomendation'>{IconUserLevel}</div>
+                      </div>
+                      <div className='Recomendation-Card--Time'>
+                        <div>10 дек 2023</div>
+                        <div>Время чтения 10мин</div>
+                      </div>
+                    </div>
+                  </SwiperSlide>
+                )
+              })
+              }
+            </Swiper>
+            <button className='SwiperArrow' ref={nextElRef2}>{arrowSlider}</button>
+          </div>
+        </div>
+      </section>
```

Comment:
I think, we can move this section to separated component. Perhaps later we will be able to reuse it in other places. If not - it's just for better readability.

Reply:
Separated compenent fix this problem